src/doc/gui_manual/Contributing/index.rst

```diff
@@ -3,31 +3,21 @@
 Contributing
 ============
 
-This is a short contributing guide on the VisIt_ project's use of
-`Sphinx <http://www.sphinx-doc.org/en/stable/tutorial.html>`_ for
-documentation.
-
-You can check out the Sphinx manual with::
-
-    svn co svn+ssh://<USERNAME>@edison.nersc.gov/project/projectdirs/visit/svn/visit/trunk/docs/SphynxDocs
-
-If you have `Sphinx <http://www.sphinx-doc.org/en/stable/tutorial.html>`_ You can
-build the html manual locally using the command::
+This is a short contributing guide on the VisIt_ project's use of `Sphinx <http://www.sphinx-doc.org/en/stable/tutorial.html>`_ for documentation.
+All of VisIt_'s documentation is found in ``<root>/src/doc``.
+There are sub-directories there for the major sub-sections of the manual including the GUI, the Python CLI, the developer's manual and more.
+If you have `Sphinx <http://www.sphinx-doc.org/en/stable/tutorial.html>`_ You can build the html manual locally using the command::
 
+    cd src/doc
     sphinx-build -b html . _build -a
 
```

Comment:
Should there be a section about being able to build them from the build tree?
if VISIT_ENABLE_MANUALS is set to true, can do 'make manuals' 
Of course, this depends on sphinx having been added to python, as is done with build_visit.

Reply:
I can be vague about the details here and just say something like...

> Depending on how your build is configured, the command ``make manuals`` may build the manual locally. If not, you can try the command ``sphinx-build -b html . _build -a`

Reply:
But one is from the build dir and one from src, correct? Will you make that distinction?

Reply:
Hmmm...good point. Not sure its necessary to tackle this nuance here.

Reply:
Tried my best.